pandas/core/reshape/reshape.py

```diff
@@ -33,6 +33,7 @@
 from pandas.core.frame import _shared_docs
 from pandas.util._decorators import Appender
 from pandas.core.index import Index, MultiIndex, _get_na_value
```

Comment:
I suspect linting will fail here, you need to remove imports that are no longer needed

Reply:
Yes, I didn't modify anything. I wanted to make sure we were on the right track first.

Reply:
ahh ok (cleanup imports here is fine though)

Reply:
Ok cleaned up imports